pkg/azurefile/azurefile.go

```diff
@@ -66,12 +70,16 @@ const (
 	// key of snapshot name in metadata
 	snapshotNameKey = "initiator"
 
-	shareNameField = "sharename"
-	diskNameField  = "diskname"
-	fsTypeField    = "fstype"
-	proxyMount     = "proxy-mount"
-	cifs           = "cifs"
-	metaDataNode   = "node"
+	shareNameField           = "sharename"
+	diskNameField            = "diskname"
+	fsTypeField              = "fstype"
+	storeAccountKeyField     = "storeaccountkey"
+	defaultSecretAccountName = "azurestorageaccountname"
+	defaultSecretAccountKey  = "azurestorageaccountkey"
+	defaultSecretNamespace   = "kube-system"
```

Comment:
`defaultSecretNamespace` should be `default`, actually it align with PVC.namespace or specified by user: https://github.com/kubernetes/kubernetes/blob/1aa2c99a27b800d237efe5e91e0223b1d1e49001/pkg/volume/azure_file/azure_provision.go#L151-L163
should add a new parameter(`secretnamespace`)

and then pass this parameter in `CreateVolume` and use that `secretnamespace` in `NodeStageVolume` func
https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/192205545d1a05bdd147bace8da3143839ff9a84/pkg/azurefile/controllerserver.go#L171-L176